### DISCUSSION TITLE
Add support for XDG base dir spec storing data in XDG_DATA_DIR

### DISCUSSION THREAD

**Discussion by jarjk** [d99kris/nchat#322] [Ideas]
IMO it would be better to store profiles and history in `~/.local/share` or `XDG_DATA_DIR` or such, because `~/.config` really only should store configs

**Comment by d99kris** [comment]
Thanks for the feedback, yes it's a valid suggestion. I will rename the topic and add a feature request tag.
Related to #171.

**Reply by jarjk** [reply]
sorry for disturbing, but any progress on it?
I mean creating an issue (feature request), just to make sure this won't be forgotten.

**Reply by d99kris** [reply]
Hi @JeromeSchmied - unfortunately there's been no progress on this feature request. If it would've been a simple fix I probably would have implemented it, but the current application logic has some assumptions w.r.t. re-initialization / setup, and then there's  backwards compatibility.

To set the expectations right, I don't anticipate I'll find time for this in a near future. I have a few other projects as well, and I usually prioritize issues/bugs. But maybe if someone else contributes a PR it can happen sooner. 🙂

**Reply by jarjk** [reply]
I see, of course this so-called feature would be a nice-to-have, but I completely understand it's no priority.
Anyways, keep up the good work!
